rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -413,7 +413,17 @@ def _wait_up_port(self, lports, wait_sync, install_method):
         if wait_sync != "none":
             ovn_nbctl.sync(wait_sync)
 
-
-
-
+    @atomic.action_timer("ovn_network.list_oflow_count_on_all_sandboxes")
+    def _list_oflow_count_on_all_sandboxes(self, sandboxes,
+                                           sandbox_args):
+        sandbox_num = len(sandboxes)
+        install_method = self.install_method
+        for j in range(sandbox_num):
+            sandbox = sandboxes[j]["name"]
+            farm = sandboxes[j]["farm"]
+            ovs_ofctl = self.farm_clients(farm, "ovs-ofctl")
+            ovs_ofctl.set_sandbox(sandbox, install_method)
+            bridge = sandbox_args.get('bridge', 'br-int')
+            lflow_count = ovs_ofctl.dump_flows(bridge)
+            LOG.debug('openflow count on %s is %s' % (sandbox, lflow_count))
```

Comment:
How does it work without return value? Just debug log?

Reply:
Yeah just debug logs for now. Can extend it to write to file?

Reply:
@jsitnicki  Ack! Will give it a try